core/scss/components/loader/placeholders.scss

```diff
@@ -133,12 +133,7 @@
           }
 
           .md-spinner-progress__mask {
-            clip: rect(
-              0px,
-              #{$circle-size},
-              #{$circle-size},
-              calc(#{$circle-size} / 2)
-            );
+            clip: rect(0px, #{$circle-size}, #{$circle-size}, calc(#{$circle-size} / 2));
```

Comment:
having some issues with the formatting, but this matches the `printWidth: 100` in the .prettierrc.json file